demos/next-11/netlify.toml

```diff
@@ -0,0 +1,15 @@
+[build]
+command = "next build"
+publish = ".next"
+ignore = "if [ $CACHED_COMMIT_REF == $COMMIT_REF ]; then (exit 1); else git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF ../..; fi;"
+
+[[plugins]]
+package = "@netlify/plugin-nextjs"
```

Comment:
I don't think this needs to be included since 

```
[[plugins]]
package = "@netlify/plugin-local-install-core"


[[plugins]]
package = "../plugin-wrapper"
```
 
is present in the file